manifests/devconsole/0.1.0/devconsole-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -135,6 +135,15 @@ spec:
           - get
           - list
           - watch
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - get
+          - create
+          - list
+          - watch
```

Comment:
just wondering, we are not watching any routes in this pr, do we still needs it?

Reply:
yes, otherwise I bumped into an rbac issue.